### ISSUE TITLE
1.2.7-dev: Support Material Overlaps on Closely Arranged Parts

### ISSUE THREAD

**Issue by simonkuehling** [alexrj/Slic3r#2819]
I just found a support material overlap issue with parts that are closely arranged together:

![3D plater](https://www.dropbox.com/s/b363ym3k20efzpq/3D-plater.png?dl=1)

Both parts are plated as separate STLs. Support material from both parts overlaps on raft layers as well as normal support layers:

![Support Preview](https://www.dropbox.com/s/jnt58tzgq0a7qgz/support-preview.png?dl=1)
![Full Preview](https://www.dropbox.com/s/0u5j50p4kevsy8k/full-preview.png?dl=1)

(Slic3r: current git master)
I suspect that a config is not necessary here - but just ask if it is of interest...

**Comment by alexrj** [comment]
That's true, support material is generated for each single object. I'm not sure about the best solution for handling this…
